sdk/tools/azure-openrewrite/src/test/resources/migrationExamples/azure-core/sample10/v2/HttpPostMultipartExample.java

```diff
@@ -0,0 +1,20 @@
+import io.clientcore.core.http.client.HttpClient;
+import io.clientcore.core.http.models.HttpHeaderName;
+import io.clientcore.core.http.models.HttpHeaders;
+import io.clientcore.core.http.models.HttpMethod;
+import io.clientcore.core.http.models.HttpRequest;
+import io.clientcore.core.models.binarydata.BinaryData;
+
+public class HttpPostMultipartExample {
+    public static void main(String... args) {
+        HttpClient client = HttpClient.getSharedInstance();
+        HttpHeaders headers = new HttpHeaders().set(HttpHeaderName.fromString("Content-Type"), "multipart/form-data");
```

Comment:
There should be an existing enum value for standard headers like Content-Type. The recipe should use those when available instead of using `fromString`.

Reply:
I'll make this change, but considering the size and importance of this PR I'd prefer to file a feature request against Azure OpenRewrite and make the change in a follow-up PR.